### PR TITLE
CFP and Supporter Tickets links

### PR DESCRIPTION
**Pull request by wolf4ood**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -87,15 +87,21 @@ <h2>Silver Sponsor{% if silver_sponsors[1] %}s{% endif %}</h2>
...
-        Sponsors
+      <a class="button primary" href="https://cfp.rustfest.eu/events/rustfest-rome-2018">
+        Submit to CFP
       </a>
+      <a class="button primary" href="https://ti.to/asquera-event-ug/rustfest-rome-2018/">
+        Supporter Tickets 
+      </a>
+      <!-- <a class="button primary" href="/sponsors/">
+        Sponsoring Information
+      </a> -->
```
Comment (skade) [inline index.html]: Any reason not to activate the sponsoring info while we are at it?

Comment (wolf4ood) [reply index.html]: @skade no reason. I was simple not sure if it was ok to activate it :)

File: index.html [hunk 2]
```diff
@@ -58,10 +58,10 @@ <h1>RustFest {{site.location.city}} {{site.dates.year}}</h1>
 {% assign gold_sponsors = site.sponsors | where: "level", "gold" %}
 {% assign silver_sponsors = site.sponsors | where: "level", "silver" %}
 
-<!--
+
 <div class='popout sponsors'>
-  <section>
-    {% if platinum_sponsors[0] %}
+ <section>
+  <!--   {% if platinum_sponsors[0] %}
```
Comment (GuillaumeGomez) [inline index.html]: This is a lot of commented lines. Shouldn't we just remove them?

File: index.html (hunk 1 above)
Comment (wolf4ood) [reply index.html]: can i activate it?

Comment (skade) [reply index.html]: Yes, though the correct URL is: /sponsors/packages/

There's a couple of tiny things I want to fix, but I can do that later.

File: index.html (hunk 2 above)
Comment (wolf4ood) [reply index.html]: It was there previously, but I guess we can remove it for now

**Review by skade — APPROVED** [review]
LGTM

**Pull request merged**